parsel/selector.py

```diff
@@ -73,12 +66,26 @@ def __getitem__(self, pos):
     def __getstate__(self):
         raise TypeError("can't pickle SelectorList objects")
 
+    def jmespath(self, query, **kwargs):
+        """
+        Call the ``.jmespath()`` method for each element in this list and return
+        their results flattened as another :class:`SelectorList`.
+
+        ``query`` is the same argument as the one in :meth:`Selector.jmespath`
+
+        Any additional named arguments are passed to the underlying
+        ``jmespath.search`` call, e.g.::
+
+            selector.jmespath('author.name', options=jmespath.Options(dict_cls=collections.OrderedDict))
+        """
+        return self.__class__(flatten([x.jmespath(query, **kwargs) for x in self]))
+
```

Comment:
```suggestion

        jmes = jmespath

```

Reply:
I think it’s better to use only one name. [I’m slightly in favor of `jmespath`](https://github.com/scrapy/parsel/issues/25#issuecomment-664984696), but I have no strong opinion about it.

Truth be told, when using `jmespath` I always import it like this:

```python
from jmespath import search as jmes
```

Reply:
I prefer `jmes` because is more in line with xpath and css (both are short). I also agree that having only one name is preferred, but in this case I think having both possibilities is not a big issue.